reference/storage/file-signUrl.md

```diff
@@ -0,0 +1,60 @@
+Create a signed url for access to file
+
+```javascript
+import { bucket, FileMode } from '@nitric/sdk';
+
+const assets = bucket('assets').for('reading', 'writing');
+
+const logo = assets.file('images/logo.png');
+
+const logoUrl = await logo.signUrl(FileMode.Read);
+```
+
+## Parameters
+---
+
+**mode** required `FileMode`
+
+The mode the signed url with operate in.
+
+Available options are:
+ - `FileMode.Read`
+ - `FileMode.Write`
+
+---
+
+**options** `SignUrlOptions`
+
+Properties
```

Comment:
I listed the object properties differently - https://github.com/nitrictech/docs/blob/0e10b06f72daf22a81f79e56300661e68e59fad0/docs/reference/api/api-route.md we should pick one style and use it in both, what do you think?

Reply:
Updated to your format.